utils/strings.py

```diff
@@ -0,0 +1,31 @@
+# Python Standard Library Imports
+from functools import reduce
+
+
+def replace(data, replacements):
```

Comment:
- If the initial value is a string, rename this param `s` instead of `data`
- The function could probably go inside `htk/utils/text/general.py`
- Maybe the function signature should be `replace_many(s, replacements)`

Reply:
The function could probably go inside `htk/utils/text/general.py`